app/views/hello/index.html.erb

```diff
@@ -2,10 +2,11 @@
 
 This is what you look like, according to your Google account:<br>
 
-<%= image_tag(@user.picture_url) %>
+<%= image_tag(@user.picture_url) %> <br>
 
-<h1>Hey, Nat. Sup?
-<br>Julia</br></h1>
-<br>Find out about us here</br>
-<%= Time.now %>
+Here's your first post: <br>
 
+<%= @post.content %> <br>
+
+
+  
```

Comment:
Got a couple of extra lines lying around here - we typically wanna make sure we put blank newlines at the end of files and remove other empty lines for cleanliness! Leave your campsite better than you found it!